data/json/items/tools.json

```diff
@@ -2426,6 +2429,7 @@
   {
     "id": "elec_chainsaw_off",
     "type": "TOOL",
+    "category": "weapon",
```

Comment:
I use chainsaws more for cutting down trees than for cutting down zombies.

Reply:
oops, that was a mistake, I got a little too happy when fixing the lajatangs.